src/components/SupportingPartners/index.js

```diff
@@ -0,0 +1,78 @@
+import { A } from "@commons-ui/core";
+import { Grid, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Section from "@/pesayetu/components/Section";
+
+const useStyles = makeStyles(({ palette, typography }) => ({
+  root: {
+    paddingTop: typography.pxToRem(56.69),
+    paddingBottom: typography.pxToRem(80),
+    background: palette.grey.light,
+  },
+  section: {},
+  title: {
+    fontWeight: 900,
+    color: palette.grey.dark,
+    marginBottom: typography.pxToRem(49.38),
+  },
+  logoContainer: {},
+  link: {},
+  logo: {
+    margin: "0",
+  },
+}));
+function SupportingPartners({ title, items, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root}>
+      <Section className={classes.section}>
+        <Typography variant="h4" className={classes.title}>
+          {title}
+        </Typography>
+        <Grid
+          container
+          spacing={4}
+          justifyContent="center"
+          className={classes.logoContainer}
+        >
+          {items?.map(({ link, logo, name }) => (
+            <Grid xs={6} md={3} lg="auto" item key={link}>
```

Comment:
```suggestion
            <Grid item key={link}>
```

Reply:
> 2\. Tablet design is off... I think we should be able to easily fit 4 logos in one row

We do not get this if I don't add the sizes there. @kilemensi

Reply:
It's beca
![Screenshot from 2021-08-16 16-05-13](https://user-images.githubusercontent.com/1779590/129568399-994d842b-8b3d-4091-ae8b-1df3d9bfdc94.png)
use of that spacing thing you love so much @KobbyMmo ... by reducing it in tablet (to 2) and not centring, I get 👇🏾 in table.